test/models/user_test.rb

```diff
@@ -0,0 +1,58 @@
+require "test_helper"
+
+class UserTest < ActiveSupport::TestCase
```

Comment:
I noticed you used RSpec for orders_controller_spec, but here you used TestCase. I think you should use the same testing library everywhere (and RSpec is preferred)